src/Network/NWContentContext.cs

```diff
@@ -126,6 +126,16 @@ public NWProtocolMetadata GetProtocolMetadata (NWProtocolDefinition protocolDefi
 			return new NWProtocolMetadata (x, owns: true);
 		}
 
+		public T GetProtocolMetadata<T> (NWProtocolDefinition protocolDefinition) where T : NWProtocolMetadata
+		{
+			if (protocolDefinition == null)
+				throw new ArgumentNullException (nameof (protocolDefinition));
+			var x = nw_content_context_copy_protocol_metadata (GetCheckedHandle (), protocolDefinition.Handle);
+			if (x == IntPtr.Zero)
+				return null;
```

Comment:
Duplicated code: `Runtime.GetINativeObject<T>` returns `null` if the handle is `IntPtr.Zero` (98% sure)